src/graphql/data/nft/Asset.ts

```diff
@@ -117,22 +116,46 @@ export function useAssetsQuery(
   last?: number,
   before?: string
 ) {
-  const queryData = useLazyLoadQuery<AssetQuery>(assetQuery, {
-    address,
-    orderBy,
-    asc,
-    filter,
-    first,
-    after,
-    last,
-    before,
+  const queryResult = useQuery(assetQuery, {
+    variables: {
+      address,
+      orderBy,
+      asc,
+      filter,
+      first,
+      after,
+      last,
+      before,
+    },
   })
-  const { data, hasNext, loadNext, isLoadingNext } = usePaginationFragment<AssetPaginationQuery, any>(
-    assetPaginationQuery,
-    queryData
-  )
+  const { loading, data, fetchMore, startPolling } = queryResult
+
+  startPolling(999)
+
+  const loadMore = () => {
+    fetchMore({
+      variables: {
+        after: data?.nftAssets?.pageInfo?.endCursor,
+      },
+    })
+  }
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      const total = (data?.nftAssets?.edges?.length || 0) + queryResult.variables?.first
+
+      queryResult?.refetch({
+        ...queryResult.variables,
+        first: total,
+      })
+    }, 999)
+
+    return () => clearInterval(intervalId)
+  }, [address, orderBy, asc, filter, first, after, last, before, queryResult.data?.nftAssets?.pageInfo?.endCursor])
+
+  const hasNext = data?.nftAssets?.pageInfo?.hasNextPage
 
-  const assets: GenieAsset[] = data.nftAssets?.edges?.map((queryAsset: { node: any }) => {
+  const assets: GenieAsset[] = data?.nftAssets?.edges?.map((queryAsset: { node: any }) => {
```

Comment:
should probably memoize this since `map` will make it a diff object every render?